render/mapping.go

```diff
@@ -45,9 +45,16 @@ func MapEndpointIdentity(m RenderableNode, local report.Networks) RenderableNode
 		return RenderableNodes{}
 	}
 
+	// We need to filter out short lived connections from this view,
+	// if they don't also have a pid; see #447
+	pid, pidOK := m.Metadata[process.PID]
+	_, conntracked := m.Metadata[endpoint.Conntracked]
+	if !pidOK && conntracked {
```

Comment:
OK, and you only squelch the node if you see it's conntracked, otherwise pseudo is created.